gpytorch/__init__.py

```diff
@@ -193,3 +227,43 @@ def test_train_mul_closure(v):
         test_test_covar = test_test_covar.sub(test_test_covar_correction)
 
     return GaussianRandomVariable(test_mean, test_test_covar), alpha
+
+
+def _variational_predict(n, full_covar, variational_mean, chol_variational_covar, alpha=None):
```

Comment:
I really don't like this here. I'm about to refactor away `_exact_predict` anyways.